rpmlint/checks/DBusPolicyCheck.py

```diff
@@ -0,0 +1,48 @@
+from xml.dom.minidom import parse
+
+from rpmlint.checks.AbstractCheck import AbstractCheck
+
+
+class DBusPolicyCheck(AbstractCheck):
+    def check(self, pkg):
+        if pkg.isSource():
+            return
+
+        files = pkg.files()
+
+        for f in files:
+            if f in pkg.ghostFiles():
+                continue
+
+            # catch xml exceptions
+            try:
+                if f.startswith('/etc/dbus-1/system.d/'):
+                    send_policy_seen = False
+                    lf = pkg.dirName() + f
+                    xml = parse(lf)
+                    for p in xml.getElementsByTagName('policy'):
```

Comment:
This is getting way too nested, could it be separated to functions please?

Reply:
Yep. Fixed.